pkgs/by-name/pr/prometheus-ebpf-exporter/package.nix

```diff
@@ -0,0 +1,76 @@
+{ lib
+, buildGoModule
+, fetchFromGitHub
+, nixosTests
+, clang
+, libbpf
+, libelf
+, libsystemtap
+, libz
+}:
+
+buildGoModule rec {
+  pname = "ebpf_exporter";
+  version = "2.4.1";
+  rev = "v${version}";
+
+  src = fetchFromGitHub {
+    inherit rev;
+    owner = "cloudflare";
+    repo = "ebpf_exporter";
+    sha256 = "sha256-quIFF0e8O/RagwqVQJrWGV8BRHErevvMJRZ3H0vbWDo=";
+  };
+
+  vendorHash = "sha256-2PokkJjsDKdM6QZq/qbZ9DOeeIWUsTbShmnop70xJhM=";
+
+  postPatch = ''
+    substituteInPlace examples/Makefile \
```

Comment:
Why exactly are examples compiled in?

Reply:
Because these are the BPF programs which make the exporter useful.